static/app/views/issueList/actions.spec.jsx

```diff
@@ -246,37 +246,6 @@ describe('IssueListActions', function () {
     expect(screen.getByRole('button', {name: 'Merge Selected Issues'})).toBeDisabled();
   });
 
-  it('disables delete and merge actions when a performance issue is selected', () => {
```

Comment:
Moved this test down with the other performance issue tests and made it check for enabled states as well to cover this case